x11-packages/far2l/build.sh

```diff
@@ -2,17 +2,17 @@ TERMUX_PKG_HOMEPAGE=https://github.com/elfmz/far2l
 TERMUX_PKG_DESCRIPTION="FAR Manager v2"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Please consider setting yourself as the package maintainer.
This field is mostly a point of contact for us to know who is working on a package.

It doesn't mean you have to do everything related to a package.